website/events/admin/views.py

```diff
@@ -291,3 +293,16 @@ def get(self, request, pk):
             "Content-Disposition"
         ] = f'attachment; filename="{slugify(event.title)}.csv"'
         return response
+
+
+@method_decorator(staff_member_required, name="dispatch")
+@method_decorator(organiser_only, name="dispatch")
+class EventMarkPresentQR(View):
```

Comment:
Do we really want this to be rendered server-side and not client-side? (For real a question, I'm not sure myself either)

Reply:
I like the simplicity of server-side here, it's more maintainable than adding yet another javascript front-end part. It also works practically straight away on any platform, making it easy for users to show/print/save/share the QR code, without needing to write maintain js for all of that.